espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.kt

```diff
@@ -43,14 +69,25 @@ class EspressoServerRunnerTest {
         }
         try {
             Server.start()
-
+            syncComposeClock.start()
             while (!Server.isStopRequestReceived) {
                 Thread.sleep(1000)
             }
         } finally {
             Server.stop()
+            syncComposeClock.join()
         }
 
         assertEquals(true, true) // Keep Codacy happy
     }
+
+    class EmptyTestRule : TestRule {
+        override fun apply(base: Statement, description: Description): Statement = base
+    }
+
+    companion object {
+        lateinit var composeTestRule: AndroidComposeTestRule<*, *>
+        val context = DriverContext()
+        const val ANDROID_ASYNC_WAIT_TIME: Long = 10
```

Comment:
I'd also add `_MS` suffix to the variable name

Reply:
changed